src/common/lib/command_mode.c

```
@@ -170,462 +170,651 @@ static uint16_t brightness_rainbow_hue = 0;      /**< Current rainbow hue (0-359
         }
-      }
 #endif
-      
-      // Use different log messages for debugging clarity
-      const char *reason;
-      if (cmd_mode.state == CMD_MODE_COMMAND_ACTIVE) {
-        reason = "Command mode timeout, returning to idle";
-      } else if (cmd_mode.state == CMD_MODE_LOG_LEVEL_SELECT) {
-        reason = "Log level selection timeout, returning to idle";
+
+        const char* reason;
```

Clang (14.0.6)

[warning] variable 'reason' is not initialized

(cppcoreguidelines-init-variables)